IslandLanding/IslandLanding/ViewModel/GameViewModel.cs

```diff
@@ -41,29 +116,35 @@ private void BackCommandExcute(object obj)
     private void ReadyCommandExcute(object obj)
     {
       IsStarting = true;
-      CalculateTimeRemaining();
+      // store time .now in varaible 
+      // when press launch compare time.now with varaible 
+      // then sub the 2 times 
+      //CalculateTimeRemaining();
```

Comment:
Is CalculateTimeRemaining() still needed here?

Reply:
no i just keep it